src/TimeView.js

```diff
@@ -56,7 +56,7 @@ var DateTimePickerTime = React.createClass({
 		return '';
 	},
 	renderDayPart: function() {
-		return DOM.div({ className: 'rdtCounter'}, [
+		return DOM.div({ className: 'rdtCounter', key: 'c'}, [
```

Comment:
It just had to be unique, I saw "c" was used elsewhere in the code, but after checking the code further, "dayPart" would be a better name for that key.